lib/src/ui/movies_page.dart

```diff
@@ -14,11 +18,225 @@ class _MoviesPageState extends State<MoviesPage> {
+                                  UiConstants.progressIndicatorStrokeWidth,
+                              backgroundColor: Colors.purple,
+                              valueColor: new AlwaysStoppedAnimation<Color>(
+                                  Colors.black),
+                            ),
+                          ),
+                        );
+                },
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _swiper(AsyncSnapshot snapshot) {
+    return Container(
+        height: UiConstants.swiperCardsHeight,
+        child: Container(
+          child: Swiper(
+            itemBuilder: (BuildContext context, int index) {
+              return new Image.network(
+                snapshot.data.results[index].posterPath,
+                fit: BoxFit.fill,
+              );
+            },
+            itemCount: snapshot.data.results.length,
+            itemWidth: UiConstants.swiperCardsWidth,
+            layout: SwiperLayout.STACK,
+          ),
+        ));
+  }
+
+  buildList(AsyncSnapshot snapshot) {
+    return snapshot.hasData
+        ? Column(
+            children: <Widget>[
+              Align(
+                alignment: Alignment.centerLeft,
+                child: Padding(
+                  padding: const EdgeInsets.only(
+                      top: UiConstants.movieTypeTextPadding,
+                      bottom: UiConstants.movieTypeTextPadding),
+                  child: new Text(
+                    'Discover movies',
+                    style: TextStyle(
+                        fontSize: UiConstants.movieTypeTitleFontSize,
+                        color: Colors.grey,
+                        fontFamily: 'Play-Bold'),
+                  ),
+                ),
+              ),
+              Container(
+                child: GridView.builder(
+                  shrinkWrap: true,
+                  physics: ScrollPhysics(),
+                  itemCount: snapshot.data.results.length,
+                  gridDelegate: new SliverGridDelegateWithFixedCrossAxisCount(
+                    mainAxisExtent: UiConstants.mainAxisExtent,
+                    crossAxisCount: UiConstants.gridColumns,
+                  ),
+                  itemBuilder: (
+                    BuildContext context,
+                    int index,
+                  ) {
+                    return GridTile(
+                      child: Container(
+                        margin: EdgeInsets.all(
+                          UiConstants.gridTileMargin,
+                        ),
+                        child: Container(
+                          child: ClipRRect(
+                            borderRadius: BorderRadius.only(
+                              topLeft: Radius.circular(
+                                  UiConstants.gridCardsBorderRadius),
+                              bottomRight: Radius.circular(
+                                  UiConstants.gridCardsBorderRadius),
```

Comment:
missed trailing commas at the end of both of the radiuses

Reply:
added